css/_welcome_page.scss

```diff
@@ -4,90 +4,70 @@ body.welcome-page {
 }
 
 .welcome {
+    background-color: $welcomePageHeaderBackground;
+    display: flex;
+    flex-direction: column;
     font-family: $welcomePageFontFamily;
-    height: 100%;
+    justify-content: space-between;
+    min-height: 100vh;
     position: relative;
 
     .header {
         align-items: center;
-        background: $welcomePageHeaderBackground;
         display: flex;
         flex-direction: column;
+        min-height: fit-content;
         overflow: hidden;
         position: relative;
         text-align: center;
 
         .header-text {
             display: flex;
             flex-direction: column;
-            justify-content: space-around;
-            margin-top: 120px;
-            margin-bottom: 20px;
+            margin-top: 156px;
+            margin-bottom: 36px;
             max-width: calc(100% - 40px);
-            min-height: 286px;
-            width: 645px;
+            width: 800px;
```

Comment:
800px seems a little bit too much for me. I'm looking at the design mockup and IMO it suggests a smaller width. I'm not sure how we end up with this number 645px previously, but when I tried 650 it looked great (given that I change the font sizes, that I believe are too big as well). Also, I believe we should definitely make this a size variable.